curriculum/challenges/english/25-front-end-development/lab-password-generator/66f53dc2c5bd6a11d6c3282f.md

```diff
@@ -82,6 +82,13 @@ assert.strictEqual(password.length, testLength);
 assert.match(password, /^[A-Za-z0-9!@#$%^&*()]+$/);
 ```
 
+You should not directly modify the globally scoped variable `password` inside of the `generatePassword` function. 
+
+```js
+const generatedPassword = generatePassword(10); 
+assert.notEqual(password,generatedPassword); 
```

Comment:
```suggestion
assert.notEqual(password, generatedPassword); 
```

if `password` was already altered in the user code, maybe by them calling the function, this test would not catch this

maybe...
```suggestion
password = '';
assert.notEqual(password, generatedPassword); 
```

Reply:
This change would cause the tests to fail, as the global password in our solution is declared as a constant.